cfpb/design-system
# Add capital-framework to repo's root, rewrite readme and contributing docs

## Changes

- [`capital-framework`](https://github.com/cfpb/capital-framework) is now in this repo's root instead of `src/`
- Purged capital framework language from both [`README.md`](https://github.com/cfpb/design-system/pull/272/files#diff-04c6e90faac2675aa89e2176d2eec7d8) and [`CONTRIBUTING.md`](https://github.com/cfpb/design-system/pull/272/files#diff-04c6e90faac2675aa89e2176d2eec7d8) and modified all the steps to include the new `@cfpb` npm scope.
- Updated the docs' website [build script](https://github.com/cfpb/design-system/pull/272/files#diff-d34809464730fe7381f1acde5e58c8e1R33) to load the capital framework stylesheets that are now in the project's root.

## Testing

Everything should work the same way it always has.

1. Pull down this branch.
1. `cd docs/`
1. `yarn`
1. `yarn start`
1. View the design system website at http://localhost:4000/design-system/

## Notes

- I have not yet changed all the CF components to use `@cfpb/` instead of `cf-`. We are still finalizing that decision with the FEWDs. Plus, it makes sense to do that at the very end of this project to avoid having to redo it if the capital-framework repo changes in the meantime.
